packages/seacas/libraries/ioss/src/Ioss_CoordinateFrame.h

```diff
@@ -9,6 +9,7 @@
 
 #include <cstdint> // for int64_t
 #include <vector>  // for vector
+#include <stdio.h>   // for printf
```

Comment:
No, not needed in the header file.

Reply:
Addressed in branch `tkordenbrock-seacas/add-ioshell-compare`.